packages/image_picker/image_picker_platform_interface/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 2.5.0
+
+* Makes `getImage` deprecated in favor of a new method `getImageFromSource`.
+* Adds `requestFullMetadata` option that allows disabling extra permission requests
+  on certain platforms.
+* Moves optional image picking parameters to `ImagePickerOptions` class.
```

Comment:
It seems like these two bullets should be nested under the one above, since they are about `getImageFromSource`. (So, indent these three lines 4 more spaces each.)